test/integration/deployment/util.go

```diff
@@ -80,7 +80,8 @@ func newDeployment(name, ns string, replicas int32) *v1beta1.Deployment {
 			Replicas: &replicas,
 			Selector: &metav1.LabelSelector{MatchLabels: testLabels()},
 			Strategy: v1beta1.DeploymentStrategy{
-				Type: v1beta1.RollingUpdateDeploymentStrategyType,
+				Type:          v1beta1.RollingUpdateDeploymentStrategyType,
+				RollingUpdate: new(v1beta1.RollingUpdateDeployment),
```

Comment:
@janetkuo We can fill the `RollingUpdate` here.

Reply:
Filling it here is fine. For the record, it is okay to set `strategy.type=RollingUpdate` while having `strategy.rollingUpdate=nil`. Default values will be filled in upon creation.